.github/workflows/dispatch.yml

```diff
@@ -14,12 +14,12 @@ jobs:
   call-build:
     name: "Build"
     needs: call-lint
-    uses: cheqd/cheqd-node/.github/workflows/build.yml@main
+    uses: cheqd/cheqd-node/.github/workflows/build.yml@DEV-1278
```

Comment:
Be careful with this on the merge. We might need to revert to `@main` once all tests are green, even though that will make the checks fail.